lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Transactions.hs

```diff
@@ -1018,6 +1019,102 @@ spec = describe "SHELLEY_TRANSACTIONS" $ do
             , expectField #expiresAt (`shouldSatisfy` isJust)
             ]
 
+    it "TRANS_MINT_01 - Mint tokens" $ \ctx -> runResourceT $ do
+      w <- fixtureWallet ctx
+
+      addrs <- listAddresses @n ctx w
+      let destination = (addrs !! 1) ^. #id
+      let payload = Json [json|{
+                              "mint_burn": {
+                                  "monetary_policy_index": "0",
+                                  "token_name": "aaaa",
+                                  "operation": {
+                                    "mint": [ [ #{destination}, { "quantity": 5, "unit": "assets" } ] ]
```

Comment:
How about?
```
  "mint": [
    {
      "address": #{destination},
      "amount": {
           "quantity": 5,
           "unit": "assets"
      }
    }
  ]
```
i.e. similar scheme as for `payments` in transactions (https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postTransaction)